modules/cluster-logging-about-crd.adoc

```diff
@@ -24,52 +18,8 @@ metadata:
   namespace: "openshift-logging" <2>
 spec:
   managementState: "Managed" <3>
-  logStore:
-    type: "elasticsearch" <4>
-    retentionPolicy:
-      application:
-        maxAge: 1d
-      infra:
-        maxAge: 7d
-      audit:
-        maxAge: 7d
-    elasticsearch:
-      nodeCount: 3
-      resources:
-        limits:
-          memory: 16Gi
-        requests:
-          cpu: "1"
-          memory: 16Gi
-      storage:
-        storageClassName: "gp2"
-        size: "200G"
-      redundancyPolicy: "SingleRedundancy"
-  visualization: <5>
-    type: "kibana"
-    kibana:
-      resources:
-        limits:
-          memory: 736Mi
-        requests:
-          cpu: 100m
-          memory: 736Mi
-      replicas: 1
-  collection: <6>
-    logs:
-      type: "fluentd"
-      fluentd:
-        resources:
-          limits:
-            memory: 736Mi
-          requests:
-            cpu: 100m
-            memory: 736Mi
+# ...
 ----
 <1> The CR name must be `instance`.
```

Comment:
For legacy method: the CR name must be `instance` .  For mCLF, It must be the same name to ClusterLogForwarder name.

Reply:
We have a separate Jira to update these that I will follow up on after: https://issues.redhat.com/browse/OBSDOCS-612